spec.md

```diff
@@ -42,10 +42,8 @@ for public re-use.
         └── original          # Intermediate folders are permitted but not necessary
             │                 # and currently contain no extra metadata.
             └── 0
-                ├── .zgroup   # Each mask itself is also a multiscale image with the extra key
-                ├── .zattrs   # "color".
-                │
-                └── t.c.z.y.x # Chunks as above.
+                ├── .zarray   # Each mask itself is a 5D array matching the highest resolution
```

Comment:
that means that we won't be able to store masks generated and a different resolution

Reply:
Not yet, no. But then ome-zarr-py code likely wouldn't support that either, nor omero-cli-zarr that output. Let's file that as an issue on one or several of the repositories and then we can work through the implications. I'm trying to get this PR into a state that it can be merged before I leave so no one is blocked on me.

Reply:
understood. 
mainly a problem I am facing with the large DBV files that I have been segmenting at a resolution that is not the highest one but we can have that in another round

Reply:
@jburel can you file an issue about the mask resolutions?